admin-dev/themes/new-theme/js/components/pagination/dynamic-paginator.ts

```diff
@@ -23,8 +23,8 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-import PaginationServiceType from '@js/types/services';
-import RendererType from '@js/types/renderers';
+import PaginationServiceType from '@js/@types/services';
```

Comment:
How come we need an `@` everywhere on all the components that are in `@js/types` ?
(I ask on this file but it's a valid question for all similar modifications)